frontend/src/photoView/photoView.js

```diff
@@ -38,7 +166,14 @@ export class PhotoView extends React.Component {
             photoData: null,
             displaySide: '',
             availableSides: [],
+            view: 0,
+            width: 1,
```

Comment:
I would probably initialize all of these values to `null` rather than `1`, bc `1` looks like real data, whereas you can check against `null` for whether these values were actually set at all. (Even though we don't do that here...)